### ISSUE TITLE
[Plugin] Can we get a list of McMMO server changes

### ISSUE THREAD

**Issue by Sabatarge** [EarthMC/Issue-Tracker#383]
This server seems to have a lot of McMMO settings that aren't the result, i've leveled a few skills now like unarmed to a high level and been told that several elements of it (including player damage) are disabled, just leveled my axes to a high level and been told that all McMMO combat damage is related. Talked to a few mods and non seem to be sure what does and doesn't work.

If we could just get a list of the changes and know what is and isn't disabled on the server it would be helpful.

Also please enable combat McMMOs, it rewards loyalty to the server

**Comment by Sabatarge** [comment]
Another thing not really worthy of another thread but taming is best gained on this server using another player (alt) as a punching bag for hours around a regen beacon since damaging other players still gives taming experience

**Comment by creatorfromhell** [comment]
Alt? Ign and alt name?

**Comment by Sabatarge** [comment]
I don't have an alt, I found out that I could level when I use to set my dogs on my friends. I'm saying that it's possible to power level it if you have one afk with armour and set your dogs

**Comment by alekb1** [comment]
Agree, it's confusing what is still a valid tactic, and what's been nerfed

**Comment by Sabatarge** [comment]
Any luck on this?

**Comment by KarlOfDuty** [comment]
I don't know what the mcmmo changes are as Fix is the one dealing with that plugin, so I can't really list them.

**Comment by Sabatarge** [comment]
Are you able to get a response from him? It's incredibly annoying to have leveled axes, unarmed and taming then finding out unarmed is disabled, axes too apparently and taming has fast food disabled

**Comment by KarlOfDuty** [comment]
I mean I don't know if he even remembers all the changes, most of them were just copied from the old emc classic server.
